client/my-sites/themes/themes-selection.jsx

```diff
@@ -377,7 +377,9 @@ export const ConnectedThemesSelection = connect(
 			// We limit the WP.org themes to one page only.
 			page: 1,
 			// WP.com theme filters don't match WP.org ones, so we add them to the search term.
-			search: filter ? `${ search } ${ filter.replace( /[+-]/g, ' ' ) }` : search,
+			search: filter
+				? `${ search } ${ filter.replaceAll( 'subject:', '' ).replace( /[+-]/g, ' ' ) }`
```

Comment:
could do with a comment explaining the replacement of +- is due to with filter values being slugs

Reply:
Also it'd be good to understand why `subject:` is a special case